Dockerfile

```diff
@@ -24,5 +24,12 @@ RUN python manage.py collectstatic --noinput
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
 RUN apt-get install -y nodejs
 
+# Install JS dependencies
+RUN npm install
```

Comment:
I don't think we need to npm install both here and in the .travis.yml.

Reply:
deleted line